sda-commons-server-morphia/src/test/java/org/sdase/commons/server/morphia/MorphiaBundleTracingIT.java

```diff
@@ -39,24 +43,43 @@
 
   @Test
   public void shouldHaveInstrumentation() {
-    Datastore datastore = getDatastore();
+    Datastore datastore = getMorphiaBundle().datastore();
     Person person = new Person().setAge(18).setName("Max");
     datastore.save(person);
 
     MockTracer tracer = getMockTracer();
-    assertThat(tracer.finishedSpans())
-        .extracting(MockSpan::operationName)
-        .contains("createIndexes", "insert");
-    assertThat(
-            tracer.finishedSpans().stream()
-                .map(s -> s.tags().keySet())
-                .flatMap(Set::stream)
-                .noneMatch(Tags.DB_STATEMENT::equals))
-        .isTrue();
+    await()
+        .untilAsserted(
+            () -> {
+              assertThat(tracer.finishedSpans())
+                  .extracting(MockSpan::operationName)
+                  .contains("createIndexes", "insert");
+              assertThat(
+                      tracer.finishedSpans().stream()
+                          .map(s -> s.tags().keySet())
+                          .flatMap(Set::stream)
+                          .noneMatch(Tags.DB_STATEMENT::equals))
+                  .isTrue();
+            });
   }
 
-  private Datastore getDatastore() {
-    return ((MorphiaTestApp) DW.getRule().getApplication()).getMorphiaBundle().datastore();
+  @Test
+  public void shouldExcludePingCommand() {
+    BasicDBObject ping = new BasicDBObject("ping", "1");
+    getMorphiaBundle().mongoClient().getDatabase(DEFAULT_DATABASE).runCommand(ping);
+
+    MockTracer tracer = getMockTracer();
+    await()
+        .pollDelay(1, TimeUnit.SECONDS)
+        .untilAsserted(
+            () ->
+                assertThat(tracer.finishedSpans())
+                    .extracting(MockSpan::operationName)
+                    .doesNotContain("ping"));
```

Comment:
(In case we need this change:) Can we also have an assertion for something that we expect to be there after the ping command? I'm not sure if the test is otherwise really testing what you want since it is hard to test that something _didn't_ happen with awaitability.